kge/model/embedder/lookup_embedder.py

```diff
@@ -86,7 +87,8 @@ def embed(self, indexes: Tensor) -> Tensor:
         return self._embed(self.embeddings(indexes.long()))
 
     def embed_all(self) -> Tensor:
-        return self._embed(self.embeddings.weight)
```

Comment:
This caused that dropout was incorrectly not applied to score funcs that call embed_all